docs/api-change-runbook.md

```diff
@@ -0,0 +1,264 @@
+# Runbook for Firecracker API changes
+
+This runbook will cover triaging API changes and ways to implement them
+appropriately.
+
+## Definitions
+
+* *Deprecated* - We will consider a deprecated API endpoint to be an endpoint
+    which still has backing functionality and can be used, but will soon be
+    removed completely along with said functionality in an upcoming version.
```

Comment:
Do we consider deprecated any endpoint that contains only a deprecated field / a couple of fields ? Or the entire endpoint has to be deprecated ? Should we also mention deprecated fields separately here ?

Reply:
Rephrased.